src/Orchard.Web/Core/Navigation/Drivers/MenuPartDriver.cs

```diff
@@ -114,5 +115,11 @@ protected override void Exporting(MenuPart part, ContentManagement.Handlers.Expo
             context.Element(part.PartDefinition.Name).SetAttributeValue("MenuText", part.MenuText);
             context.Element(part.PartDefinition.Name).SetAttributeValue("MenuPosition", part.MenuPosition);
         }
+
+        protected override void Cloning(MenuPart originalPart, MenuPart clonePart, CloneContentContext context) {
+            clonePart.MenuText = originalPart.MenuText;
+            clonePart.MenuPosition = originalPart.MenuPosition;
+            clonePart.Menu = context.CloneContentItem;
```

Comment:
`context.CloneContentItem` causes the "Show on a menu" to always be checked incorrectly.
```suggestion
            clonePart.Menu = originalPart.Menu;
```